documentation/contributing/contributor-guide.adoc

```diff
@@ -113,12 +122,12 @@ $ git push -f origin HEAD
 ----
 . Visit your fork on GitHub
 . Click *Compare & pull request*
-. If a SME review is needed:
-.. Add *SME Review Needed* label.
-.. When SME review is complete, remove *SME Review Completed* label and go to step 16.
-. Add *Peer Review Needed* label and assign a Reviewer.
-.. When peer review has been completed, the reviewer deletes the *Peer Review Needed* label and adds the *Peer Review Done* label.
-. When *Peer Review Done* label has been added:
-.. If no changes are needed, remove *Peer Review Done* label and add *Ready for Merge* label.
-.. If changes are needed and the changes are minor, make changes to the files, commit, and push, and then remove *Peer Review Done* label and add *Ready for Merge* label.
-.. If you want an additional review on the updated content, add the *Peer Review Needed* label and ask for another review in a comment.
+. If an SME review is needed:
+.. Add the *SME Review Needed* label.
+.. When the SME review is complete, add the *SME Review Done* label and go to step 16.
+. Add the *Peer Review Needed* label to assign a reviewer.
+.. When a peer review is complete, the reviewer removes the *Peer Review Needed* label and adds the *Peer Review Done* label.
+. When the *Peer Review Done* label has been added:
+.. If no changes are needed, remove the *Peer Review Done* label and add the *Ready for Merge* label.
+.. If changes are needed and the changes are minor, make changes to the files, commit, and push, and then remove the *Peer Review Done* label and add the *Ready for Merge* label.
+.. If you want an additional review on the updated content, add the *Peer Review Needed* label again, and ask for another review in a comment.
```

Comment:
Again, I'm not sure if everyone can set labels. I think this is technically correct, but would rather confuse any contributors. Maybe it is better if they just open the PR and we take care of the rest.

Reply:
Yes. I've dropped all this. I've stated in the intro that reviewers might be assigned.